xdsl/irdl.py

```diff
@@ -1289,6 +1282,14 @@ def irdl_definition(cls: type[_OpT]):
 
     new_attrs["irdl_definition"] = irdl_definition
 
+    custom_verify = getattr(cls, 'verify_')
+
+    def verify_(self: _OpT):
+        op_def.verify(self)
+        custom_verify(self)
```

Comment:
I think you could just make this a method of IRDLOperation now, instead of generating it in irdl_op_definition?